deploy.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: secrethub-mutating-webhook
+  labels:
+    app: secrethub-mutating-webhook
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: secrethub-mutating-webhook
+  template:
+    metadata:
+      labels:
+        app: secrethub-mutating-webhook
+    spec:
+      containers:
+      - name: secrethub-mutating-webhook
+        image: simonbarendse/kubernetes-mutating-webhook:latest
+        volumeMounts:
+        - name: webhook-certs
+          mountPath: /etc/webhook/certs
+          readOnly: true
+      volumes:
+      - name: webhook-certs
+      initContainers:
+        - name: secrethub
+          image: secrethub/cli:0.38.0
+          env:
+            - name: SECRETHUB_CREDENTIAL
+              valueFrom:
+                secretKeyRef:
+                  name: secrethub-mutating-webhook
+                  key: secrethub_credential
+          volumeMounts:
+          - name: webhook-certs
+            mountPath: /webhook-certs
+            readOnly: false
+          command:
+            - /bin/sh
+            - -c
+          args:
+            - "secrethub read kubernetes-webhook/webhook/cert.pem -o /webhook-certs/cert.pem && secrethub read kubernetes-webhook/webhook/key.pem -o /webhook-certs/key.pem"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: secrethub-mutating-webhook
+  labels:
+    app: secrethub-mutating-webhook
+spec:
+  type: LoadBalancer
+  ports:
+  - port: 443
+    targetPort: 443
+  selector:
+    app: secrethub-mutating-webhook
+---
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: secrethub-mutating-webhook
+  labels:
+    app: secrethub-mutating-webhook
+    kind: mutator
+webhooks:
+- name: secrethub-mutating-webhook.default.svc.cluster.local
+  clientConfig:
+    service:
+      name: secrethub-mutating-webhook
+      namespace: default
+      path: "/"
+    caBundle: "${CA_BUNDLE}"
```

Comment:
This should be replaced with the CA bundle of your cluster, which can be obtained with `kubectl config view --raw --minify --flatten -o jsonpath='{.clusters[].cluster.certificate-authority-data}'`